### ISSUE TITLE
busy() not implemented

### ISSUE THREAD

**Issue by mje-nz** [PaulStoffregen/WS2812Serial#4]
The WS2812Serial::busy() method is not implemented. It'd be nice if it was the "wait if prior DMA still in progress" and "wait 300us WS2812 reset time" from show(). I'm happy to send a PR if you want.

**Comment by PaulStoffregen** [comment]
> I'm happy to send a PR if you want.

Yes, please do.  I'm fully consumed with porting USB host code right now.
